lib/util.js

```diff
@@ -885,6 +887,13 @@ function formatRaw(ctx, value, recurseTimes) {
   }
   ctx.seen.pop();
 
+  if (ctx.sorted && extrasType === kObjectType) {
+    if (ctx.sorted === true) {
+      output = output.sort();
+      return reduceToSingleString(ctx, output.sort(), base, braces);
+    }
```

Comment:
☝️ This is sorting things twice. I believe the first one `output = output.sort()` isn't needed.